cmd/ctr/commands/tasks/metrics.go

```diff
@@ -166,6 +162,34 @@ func printCgroup2MetricsTable(w *tabwriter.Writer, data *v2.Metrics) {
 	}
 }
 
+func printWindowsStats(w *tabwriter.Writer, windowsStats *wstats.Statistics) error {
```

Comment:
This is a bit of hack due to hcsshim <-> containerd cyclic dependencies and the fact that hcsshim uses gogo/protobuf.